library/std/src/net/ip.rs

```diff
@@ -3,12 +3,44 @@
 mod tests;
 
 use crate::cmp::Ordering;
-use crate::fmt::{self, Write as FmtWrite};
-use crate::io::Write as IoWrite;
+use crate::fmt::{self, Write};
 use crate::mem::transmute;
+use crate::str;
 use crate::sys::net::netc as c;
 use crate::sys_common::{FromInner, IntoInner};
 
+/// Used for slow path in `Display` implementations when alignment is required.
+struct IpDisplayBuffer<const SIZE: usize> {
+    buf: [u8; SIZE],
```

Comment:
This could (should?) be `MaybeUninit<u8>` instead.

Reply:
Implemented using `MaybeUninit<u8>` now.